crates/subspace-networking/examples/requests.rs

```diff
@@ -22,20 +34,9 @@ async fn main() {
         }),
         allow_non_globals_in_dht: true,
         request_response_protocols: vec![
-            PiecesByRangeRequestHandler::create(|req| {
```

Comment:
Why do we need a mock request here instead of the pieces-by-range request?

Reply:
I just thought this example is not particular request related, so we can use any request here.

Reply:
I don't see changes for `ObjectMappingsRequestHandler`. Am I missing something?

Reply:
Yeah, you right. Sorry, didn't commit it 😅